conf/machine/colibri-imx7-emmc.conf

```diff
@@ -0,0 +1,31 @@
+#@TYPE: Machine
+#@NAME: Toradex Colibri iMX7 Dual 1GB (eMMC)
+#@SOC: i.MX 7Dual
+#@DESCRIPTION: Machine configuration for Toradex Colibri iMX7 SOM
+#@MAINTAINER: Stefan Agner <stefan.agner@toradex.com>
+
+MACHINEOVERRIDES =. "mx7:mx7d:"
+
+include conf/machine/include/imx-base.inc
+include conf/machine/include/tune-cortexa7.inc
+
+PREFERRED_PROVIDER_virtual/kernel ?= "linux-toradex"
+KERNEL_IMAGETYPE = "zImage"
+KERNEL_DEVICETREE += "imx7d-colibri-emmc-eval-v3.dtb imx7d-colibri-emmc-aster.dtb"
+
+# The kernel lives in a seperate FAT partition, don't deploy it in /boot/
+RDEPENDS_${KERNEL_PACKAGE_NAME}-base = ""
+
+PREFERRED_PROVIDER_u-boot ?= "u-boot-toradex"
+PREFERRED_PROVIDER_virtual/bootloader ?= "u-boot-toradex"
+PREFERRED_PROVIDER_u-boot-fw-utils ?= "u-boot-toradex-fw-utils"
+UBOOT_MACHINE ?= "colibri_imx7_emmc_defconfig"
+
+IMAGE_FSTYPES += "wic.gz tar.xz"
+# wic support
+WKS_FILE = "sdimage-bootpart.wks"
+WIC_CREATE_EXTRA_ARGS_append = " --no-fstab-update"
+
```

Comment:
These three lines are sort of provided by imx-base.inc and could be deleted.
As we fuse the SoM to boot from the on module eMMC, first boot area U-Boot is not needed as part of the wic image, however it does also not hurt, so we could switch to the imx-base.inc setting "sdimage-bootpart.wks".

Reply:
In fact, adding the U-Boot adds an extra dependency and is technically wrong as it is unused. I'd prefer to not write it there due to that.

Reply:
I am fine with keeping the WKS_FILE here.